doc/news/changes.h

```diff
@@ -366,6 +366,12 @@ inconvenience this causes.
 
 <ol>
 
+  <li> New: set_all_manifold_ids_on_boundary(boundary_id, manifold_id) 
+  which sets the manifold_id to the given boundary_id.
```

Comment:
The section is ok, but please add the classname here and omit the argument list. I think you want to say this:
```
<li> New: Triangulation::set_all_manifold_ids_on_boundary() 
which sets the manifold_id for all parts of the boundary with a given boundary_id.
```